buildSrc/src/main/java/org/robolectric/gradle/GradleManagedDevicePlugin.kt

```diff
@@ -0,0 +1,37 @@
+package org.robolectric.gradle
+
+import com.android.build.api.dsl.LibraryExtension
+import com.android.build.api.dsl.ManagedVirtualDevice
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.configure
+import org.gradle.kotlin.dsl.create
+
+class GradleManagedDevicePlugin : Plugin<Project> {
+  override fun apply(project: Project) {
+    project.pluginManager.apply("com.android.library")
```

Comment:
The idea is to make `GradleManagedDevicePlugin` autonomous. By adding this, we know we can apply this plugin, and it should just work.
But since it worked before without this, I can remove it if you prefer.

Reply:
Good idea, I've updated the PR accordingly.
Now the plugin checks if the `com.android.library` plugin is applied and displays a warning if it is not.